### ISSUE TITLE
Accessibility - Video Player - Screen reader - Focus order in the video player 

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1773]
## Detailed description

Issue :- 
1. the focus order is not logical. Example: the picture in picture button is identified before the play button in arrow key navigation. where as, in tab navigation, it is announced just before the unlabelled link at the end of the frame.
2. In tab/shift tab navigation, After full screen button there are Two tab stops where Screen reader does not announce anything.
Note: Because of the above mentioned issues in the video player, the Keyboard navigation is not intuitive for the user.

Expected -
Make the Keyboard focus logical and intuitive for the user.

**Comment by RaphaelAmadeu** [comment]
@RobertaJHahn 
This issue is from Kaltura videoplayer, we are just using it as provided by the Mediacenter team, so this fix must be addressed by them, Thank you!
